lib/Tool/Requirements.php

```diff
@@ -609,10 +609,18 @@ public static function checkPhp()
             ]);
         }
 
+        $utfLocaleState = Check::STATE_ERROR;
+        if (
+            setlocale(LC_ALL, ['en.utf8', 'en_US.utf8', 'en_GB.utf8'])
```

Comment:
This only checks the requirement. the actual usage is here https://github.com/pimcore/pimcore/blob/83142b53cfd5a791e3c6167470a66336fd03da22/lib/Console/Application.php#L59